colabs/Similarity_tests.ipynb

```diff
@@ -0,0 +1,1113 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "DUjhhU73FnVz",
+        "outputId": "7eb1b182-827d-49f6-d237-82b3cefd9a2d"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Mounted at /content/gdrive\n"
+          ]
+        }
+      ],
+      "source": [
+        "from google.colab import drive\n",
+        "drive.mount('/content/gdrive')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!git clone https://github.com/PiotrMaciejKowalski/BigData2022-actors.git\n",
+        "!mv /content/BigData2022-actors/* .\n",
+        "!mv /content/BigData2022-actors/.* .\n",
+        "!rmdir /content/BigData2022-actors/"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "RsNaC_WnFtrT",
+        "outputId": "f92eaf3f-c619-481e-d90e-14c03e53267e"
+      },
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Cloning into 'BigData2022-actors'...\n",
+            "remote: Enumerating objects: 1926, done.\u001b[K\n",
+            "remote: Counting objects: 100% (547/547), done.\u001b[K\n",
+            "remote: Compressing objects: 100% (179/179), done.\u001b[K\n",
+            "remote: Total 1926 (delta 469), reused 415 (delta 368), pack-reused 1379\u001b[K\n",
+            "Receiving objects: 100% (1926/1926), 6.11 MiB | 12.33 MiB/s, done.\n",
+            "Resolving deltas: 100% (1201/1201), done.\n",
+            "mv: cannot move '/content/BigData2022-actors/.' to './.': Device or resource busy\n",
+            "mv: cannot move '/content/BigData2022-actors/..' to './..': Device or resource busy\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# !git checkout Wyjanienie-modelu---analiza-bdw"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "pUQ7CPOuFtoc",
+        "outputId": "e5f1ed26-d65c-4a35-891f-0c5c8d6eb707"
+      },
+      "execution_count": 3,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Branch 'Wyjanienie-modelu---analiza-bdw' set up to track remote branch 'Wyjanienie-modelu---analiza-bdw' from 'origin'.\n",
+            "Switched to a new branch 'Wyjanienie-modelu---analiza-bdw'\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!chmod +x setup_sparka.sh\n",
+        "!./setup_sparka.sh"
+      ],
```

Comment:
to chyba też jest niepotrzebne. spark był potrzebny do preprocessingu, a tutaj już czytamy zapisane dane

Reply:
no tak, bo nie używam już spark - usunęłam